releases/release-6.5.3.md

```diff
@@ -0,0 +1,96 @@
+---
+title: TiDB 6.5.3 Release Notes
+summary: Learn about the compatibility changes, improvements, and bug fixes in TiDB 6.5.3.
+---
+
+# TiDB 6.5.3 Release Notes
+
+Release date: xxx, 2023
+
+TiDB version: 6.5.3
+
+Quick access: [Quick start](https://docs.pingcap.com/tidb/v6.5/quick-start-with-tidb) | [Production deployment](https://docs.pingcap.com/tidb/v6.5/production-deployment-using-tiup) | [Installation packages](https://www.pingcap.com/download/?version=v6.5.3#version-list)
+
+## Compatibility changes
+
+- note 1
+
+## Improvements
+
++ TiDB
+
+    - note 1
+
++ TiKV
+
+    - note 1
+
++ PD
+
+    - note 1
+
++ TiFlash
+
+    - note 1
+
++ Tools
+
+    + Backup & Restore (BR)
+
+        - note 1
+
+    + TiCDC
+
+        - note 1
```

Comment:
```suggestion
        - Optimize the way TiCDC handles DDLs so that DDLs do not block the use of other unrelated DML Events, and reduce memory usage [#8106](https://github.com/pingcap/tiflow/issues/8106) [asddongmen](https://github.com/asddongmen)
        - Optimize the Decoder interface and add a new parameter `AddKeyValue` [#8861](https://github.com/pingcap/tiflow/issues/8861) [3AceShowHand](https://github.com/3AceShowHand)
```